index-lt-4.7.4.d.ts

```diff
@@ -0,0 +1,17 @@
+export * from './index';
+
+declare global {
```

Comment:
Nit: a comment here about why this is needed and when we should consider cleaning it up would be helpful.